docs/reference/query-dsl/nested-query.asciidoc

```diff
@@ -114,4 +116,125 @@ If `false`, {es} returns an error if the `path` is an unmapped field.
 
 You can use this parameter to query multiple indices that may not contain the
 field `path`.
---
\ No newline at end of file
+--
+
+[[nested-query-notes]]
+==== Notes
+
+[[multi-level-nested-query-ex]]
+===== Multi-level nested queries
+
+To see how multi-level nested queries work,
+try the following example.
+
+First, create an index, `drivers`,
+with a nested field mapping.
+
+[source,console]
+----
+PUT /drivers
+{
+    "mappings" : {
+        "properties" : {
+            "driver" : {
+                "type" : "nested",
+                "properties" : {
+                    "vehicle" : {
+                        "type" : "nested",
+                        "properties" : {
+                            "model" : {                 <1>
+                                "type" : "keyword"
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+----
+
+<1> Note the `model` field is nested as follows:
```

Comment:
Instead of a callout, I would incorporate this information into the preceding text. Something along the lines of:

```To see how multi-level nested queries work, first you need an index that has nested fields. The following request defines mappings for the `drivers` index with the `model` field nested under `driver -> vehicle -> model`:```